index.js

```diff
@@ -344,6 +357,45 @@ class ScmBase {
     stats() {
         return {};
     }
+
+    /**
+     * Get an array of scm context (e.g. [github.com, mygitlab_gitlab])
+     * @method getScmContexts
+     * @return {Array}
+     */
+    getScmContexts() {
+        return validate(this._getScmContexts(), Joi.array().items(
```

Comment:
`getScmContexts()` should return array value, so can not use this validate method here.
This vlidate method returns Promise object.